boards/kyria/main.py

```diff
@@ -1,42 +1,67 @@
 from kyria_v1_rp2040 import KMKKeyboard
 
-from kmk.extensions.media_keys import MediaKeys
-from kmk.extensions.rgb import RGB, AnimationModes
+from kmk.hid import HIDModes
 from kmk.keys import KC
-from kmk.modules.encoder import EncoderHandler
 from kmk.modules.layers import Layers
 from kmk.modules.modtap import ModTap
 from kmk.modules.split import Split, SplitType
+from kmk.modules.tapdance import TapDance
+from kmk.modules.encoder import EncoderHandler
+from kmk.extensions.rgb import RGB, AnimationModes
+from kmk.extensions.media_keys import MediaKeys
+from kmk.extensions.oled import (
+    Oled,
+    OledData,
+)
+from kmk.extensions.international import International
 
 keyboard = KMKKeyboard()
 keyboard.debug_enabled = True
 
-keyboard.modules.append(Layers())
-keyboard.modules.append(ModTap())
-keyboard.extensions.append(MediaKeys())
+encoder_handler = EncoderHandler()
+encoder_handler.pins = ((keyboard.encoder_pin_0, keyboard.encoder_pin_1, None, False),)
+
+keyboard.modules = [Layers(), ModTap(), TapDance()]
+keyboard.extensions = [MediaKeys(), International()]
 
-# Using drive names (KYRIAL, KYRIAR) to recognize sides; use split_side arg if you're not doing it
 split = Split(split_type=SplitType.UART, use_pio=True)
 keyboard.modules.append(split)
 
-# Uncomment below if you're using encoder
-encoder_handler = EncoderHandler()
-encoder_handler.pins = ((keyboard.encoder_pin_0, keyboard.encoder_pin_1, None, False),)
-
-# Uncomment below if you're having RGB
 rgb_ext = RGB(
     pixel_pin=keyboard.rgb_pixel_pin,
     num_pixels=10,
+    val_limit=200,
+    val_default=20,
     animation_mode=AnimationModes.BREATHING_RAINBOW,
 )
 keyboard.extensions.append(rgb_ext)
 
+oled_ext = Oled(
+    OledData(
+        entries=[
```

Comment:
Don't have a good name for these. Supports "string with a position and layer" and "image with a position and layer", whatever to call those.